code/src/Components/HappyThoughts.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react"
+
+import moment from "moment"
+
+import "./HappyThoughts.css"
+
+export const HappyThoughts = props => {
+    const {message, hearts, createdAt} = props.thought
+    const [likes, setLikes] = useState(hearts)
+    const handleClick = () => {
+        fetch ("https://happy-thoughts-technigo.herokuapp.com/thoughts", {
```

Comment:
This is not proper endpoint (URL) for updating likes count. Please, check out project brief once again and provide changes

Reply:
That was a big mistake, sorry! I fixed it!